eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -1416,6 +1417,9 @@ getUTxO = getsNES utxoL
 getProtVer :: EraGov era => ImpTestM era ProtVer
 getProtVer = getsNES $ nesEsL . curPParamsEpochStateL . ppProtocolVersionL
 
+setProtVer :: EraGov era => ProtVer -> ImpTestM era ()
+setProtVer pv = modifyNES $ nesEsL . curPParamsEpochStateL . ppProtocolVersionL .~ pv
```

Comment:
It would be nice to add a changelog entry for this addition. Also, it is incorrect to only modify current PParams, because of the rotation mechanism, which will clear out those changes on an epoch boundary.
```suggestion
setProtVer pv = modifyPParams $ ppProtocolVersionL .~ pv
```